routes/admin.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose')
+const express = require('express')
+const { User, fullName } = require('../models/user')
+const router = express.Router()
+
+router.get('/', async (req, res) => {
+  const users = await User.find()
+
+  res.send(users)
+})
+
+router.get('/owners', async (req, res) => {
+    const owners = await User.find(
+        { "role": /hotelOwner/i }
```

Comment:
hotelOwner should not be used like this, we have roles.js in models directory so use it